1.16.39

```diff
@@ -0,0 +1,14 @@
+Requirement already satisfied: boto3 in /home/debo/miniconda3/lib/python3.8/site-packages (1.16.39)
```

Comment:
Why do we need this file?

Reply:
removed as well.may be generated due to devdocker extension in vscode